lldb/unittests/ScriptInterpreter/Python/PythonTestSuite.cpp

```diff
@@ -9,43 +9,27 @@
 #include "gtest/gtest.h"
 
 #include "Plugins/ScriptInterpreter/Python/SWIGPythonBridge.h"
-#include "Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.h"
-#include "Plugins/ScriptInterpreter/Python/ScriptInterpreterPythonImpl.h"
 #include "Plugins/ScriptInterpreter/Python/lldb-python.h"
 
-#include "lldb/Host/FileSystem.h"
-#include "lldb/Host/HostInfo.h"
-
 #include "PythonTestSuite.h"
 #include <optional>
 
-using namespace lldb_private;
-class TestScriptInterpreterPython : public ScriptInterpreterPythonImpl {
-public:
-  using ScriptInterpreterPythonImpl::Initialize;
-};
-
 void PythonTestSuite::SetUp() {
-  FileSystem::Initialize();
```

Comment:
This is perhaps the issue?

Reply:
FWIW, I think it's appropriate that `FileSystem::Initialize()` is removed here, because `PythonTestSuite` itself does not seem to interact w/ `FileSystem`. But `PythonDataObjectsTests.cpp` does, so it should be the one to take care of `FileSystem::Initialize()`.